### DISCUSSION TITLE
Upgrade (path) from old release to new release

### DISCUSSION THREAD

**Discussion by seimic** [cockroachdb/cockroach#132050] [Q&A] [answered]
Hi,

could you tell me if it's safe to upgrade a minor version, lets say v23.1.11, to the latest minor release v23.1.27 without to
miss any migration steps in the minor releases in between?
I suppose its the recommended upgrade procedure to upgrade to the highest minor release first before upgrading to the
next major release, right?

Thanks,
Michael

**Comment by yuzefovich** [comment, marked as answer]
Yes, you can skip intermediate minor releases and go from v23.1.11 straight to v23.1.27. You also could upgrade from v23.1.11 to v23.2.latest, but for major upgrades we _do_ recommend upgrading to the latest minor first.
